ui/app/css/rules.less

```diff
@@ -0,0 +1,8 @@
+.rules-filters__label {
```

Comment:
I appreciate what you're trying to do with these css changes, but I think they actually take away from the page a bit. Users are primarily focused on the rules themselves -- adding all the extra whitespace/size to the controls takes away from that. These controls are sometimes sometimes used right after the page loads, but most of the time spent on the page is down in the rules themselves -- so keeping the focus on them is preferable.